src/main/java/pages/BasePage.java

```diff
@@ -121,11 +121,11 @@ protected void tapByCoordinates(int x, int y) {
     }
 
     public int getXPositionForElement(String element) {
-        return findElementByXpath("//*[@name='" + element + "']").getLocation().getX();
+        return findElementByXpath("//XCUIElementTypeButton[contains(@name, '" + element + "')]").getLocation().getX();
     }
 
     public int getYPositionForElement(String element) {
-        return findElementByXpath("//*[@name='" + element + "']").getLocation().getY();
+        return findElementByXpath("//XCUIElementTypeButton[contains(@name, '" + element + "')]").getLocation().getY();
```

Comment:
please make this generic as it could have high impact on other elements

Reply:
added * instead